src/QsFmt/Formatter/SyntaxTree/Statement.fsi

```diff
@@ -21,6 +21,31 @@ type internal SymbolBinding =
     /// A declaration for a tuple of new symbols.
     | SymbolTuple of SymbolBinding Tuple
 
+type internal QubitSymbolBinding =
+    | QubitSymbolDeclaration of Terminal
+    | QubitSymbolTuple of QubitSymbolBinding Tuple
+
+module internal QubitSymbolBinding =
+    /// <summary>
+    /// Maps a qubit symbol binding by applying <paramref name="mapper"/> to its leftmost terminal's trivia prefix.
+    /// </summary>
+    val mapPrefix: mapper:(Trivia list -> Trivia list) -> QubitSymbolBinding -> QubitSymbolBinding
+
+type internal SingleQubit = { Qubit: Terminal; OpenParen: Terminal; CloseParen: Terminal }
```

Comment:
Reminder to myself to add documentation comments here, once the dust settles a bit on the data types to be used.

Reply:
done